### ISSUE TITLE
Looking forward for react hook support.

### ISSUE THREAD

**Issue by dwsyoyo** [aweary/react-copy-write#62]
It is the time to update the Consumer to react hook.

**Comment by davej** [comment]
Yes. I don't think there's a reason why render props can't also still be supported.

**Comment by davej** [comment]
I've added a PR that introduces react hook support. #63.

Looks like there hasn't been much activity on this repo lately so I'm not sure if it will get merged. If not, then I'll create a fork and continue working on this.

@aweary: Pinging you for an update on the status of this project.
